src/lib/withAllLocales.js

```diff
@@ -0,0 +1,9 @@
+import withLocales from 'cozy-ui/transpiled/react/I18n/withLocales'
+
+const dictRequire = lang => {
+  const cozyClientLocales = require(`cozy-client/dist/models/doctypes/locales/${lang}.json`)
+  const cozySettingsLocales = require(`../locales/${lang}.json`)
+  return { CozyClient: cozyClientLocales, ...cozySettingsLocales }
+}
+
+export default withLocales(dictRequire)
```

Comment:
Could we provide a little bit of documentation here?

I am not sure everybody can understand what does this helper.

I would use jsdoc to do that.

```suggestion
/**
 * @function
 * @description HOC to provide locales from CozyClient and from the application to components.
 * @param  {Function} - Component that need translations from CozyClient
 * @returns {Function} - Component that will receive translations from CozyClient and CozySettings
 */
export default withLocales(dictRequire)
```